lib/rbac/filterer.rb

```diff
@@ -557,6 +561,8 @@ def scope_targets(klass, scope, rbac_filters, user, miq_group)
 
         filtered_ids = calc_filtered_ids(associated_class, rbac_filters, user, miq_group, scope_tenant_filter)
         scope_by_parent_ids(associated_class, scope, filtered_ids)
+      elsif apply_rbac_by_scope_in?(klass)
```

Comment:
1. Is this generic logic? This is saying that we will never want to show button events to any user, ever.

Reply:
1. as I know from @skateman, it is true.

2. I can use this to implement own scope for MiqSchedule as I stated in description.

Reply:
I got the info from @h-kataria that we want these things for admin and super admin users only.